paf-mvp-operator-client-express/src/operator-backend-client.ts

```diff
@@ -130,7 +131,7 @@ export class OperatorBackendClient {
       return undefined;
```

Comment:
should set `lastRefresh` cookie line 120, no?

Reply:
Ouch, I was surprised to learn we set cookies from URI data on the server-side. And on the frontend side as well? Why do we need duplicated functionality? And why pifmarket and pofmarket do not use this middleware? @OlivierChirouze

Reply:
Yes, I understand this is confusing.
The idea to set it server-side is that it can be more robust than to set it front-side (due to browser restrictions).
Now, this backend component is optional, this is why other advertisers don't use it in the demo.

Reply:
Ok. Done